Makefile

```
@@ -1,7 +1,29 @@
+scratch-gui-version = v3.2.37
+
 .PHONY: init
 init:
 	npm install --include=dev
 	brew install clang-format
+	make scratch-gui-install
+
+.PHONY: scratch-gui-install
+scratch-gui-install:
+	git clone --depth 1 --branch $(scratch-gui-version)  https://github.com/scratchfoundation/scratch-gui.git
+	cd scratch-gui && npm install
+	make scratch-gui-patch
+	
+.PHONY: scratch-gui-patch
+scratch-gui-patch:
+	cd scratch-gui && node ../scratch-arduino-extensions/scripts/patch-gui.js
+	cd scratch-gui && node ../scratch-arduino-extensions/scripts/add-banner.js
+
+.PHONY: scratch-gui-start
+scratch-gui-start:
+	cd scratch-gui && npm start
+
```

we could add the `watch` command 

```suggestion
.PHONY: scratch-gui-watch
scratch-gui-watch:
	cd scratch-gui && npm watch
```